src/Evaluator.ts

```diff
@@ -107,6 +107,18 @@ export default class Evaluator {
     this.layerOverrides[layerName] = overrides;
   }
 
+  public clearAllGateOverrides(): void {
```

Comment:
I had this PR to jest that was like:

- https://github.com/jestjs/jest/pull/12460

but the basic theme was like, do you want to either:

1. Just reset state automatically in some afterEach/afterAll hook (which is inefficient but actual real world impact small)
2. Have something that only runs in an afterAll hook that blows up if you see overrides that havent been unset

To get 2 working, you need to be able to either check if overrides exist, or just get the overrides, and then can have an afterAll hook that's like "yo you set this override but forgot to clear it". Idk if that makes sense but I'm wondering how you plan to use these, and how we plan to detect if someone forgot to use it.

Reply:
TL;DR should we have a method to be able to see if there are overrides so that you can have a `beforeEach` hook like:

`if overridesOn then clearOverrides`

but from our offline conversation, `clearOverrides` should be so fast that there's no point in adding the overhead